src/test/java/fr/insee/pearljam/api/authKeycloak/TestAuthKeyCloak.java

```diff
@@ -155,7 +154,7 @@ private ResultMatcher expectTimestampFromCurrentDate(String expression, int unit
 	 * Test that the GET endpoint "api/user"
 	 * return 200
 	 * 
-	 * @throws Exception
+	 * @throws Exception e
 	 */
 	@Test
 	@Order(1)
```

Comment:
maybe we should 
- get rid of orders
- have scripts to reinit database content used when updates are done
- use test containers